03-task-manager/starter/db/connect.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose')
+
+const connectDB = async(url) =>{
+   try {
+        await mongoose.connect(url, {
+        useNewUrlParser: true,
+        useCreateIndex : true,
+        useCreateIndex: true,
+        useFindAndModify: false,
+        useUnifiedTopology: true,
```

Comment:
Once/if you update your Mongoose version, you will want to remove these options (as they'll no longer work and throw errors and/or warnings):
`await mongoose.connect(url, {});`